webapp/src/main/webapp/WEB-INF/portlet.xml

```diff
@@ -1115,4 +1115,23 @@
     </portlet-info>
   </portlet>
 
+  <portlet>
+    <portlet-name>SimpleStorage</portlet-name>
+    <display-name>Simple storage portlet</display-name>
+    <portlet-class>org.exoplatform.commons.api.portlet.GenericDispatchedViewPortlet</portlet-class>
+    <init-param>
+      <name>portlet-view-dispatched-file-path</name>
+      <value>/html/simpleStorage.html</value>
+    </init-param>
```

Comment:
When it's an HTML (static response), it would be better to add a cache in order to not `render` the html file each time the portlet is called.

```suggestion
    </init-param>
    <expiration-cache>-1</expiration-cache>
    <cache-scope>PUBLIC</cache-scope>
```

Reply:
Done